daemon/daemon.go

```diff
@@ -547,7 +607,7 @@ func (d *Daemon) Stop(sigCh chan<- os.Signal) error {
 		d.snapListener.Close()
 	}
 
-	if restartSystem {
+	if restarting {
```

Comment:
this is a change, no?  we werent' doing this before RestartDaemon or RestartSocket

Reply:
yes you're correct this is different, I fixed this now